pzsd_bot/cogs/misc/dice.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from random import randint
+
+import discord
+from discord import ApplicationContext, Bot
+from discord.commands import SlashCommandGroup, option
+from discord.ext.commands import Cog
+
+from pzsd_bot.settings import DiceSettings, Emoji
+
+logger = logging.getLogger(__name__)
+
+D6_FACES = {
```

Comment:
why not just make these arrays?

Reply:
Oh true, I suppose I could just make this a tuple

Reply:
yeah I think it makes more sense. You could even dynamically get the property and remove these, but maybe that's too cryptic

Reply:
`D20_FACES = {i: getattr(Emoji, f"d20_{i}") for i in range(1, 21)}`

Reply:
I forget whether range is inclusive or not